temp_tests.py

```diff
@@ -0,0 +1,236 @@
+import traceback
```

Comment:
Presumably this will be changed to use unittest and be moved to a proper test file so that it runs with the CI?